src/main/java/ru/skypro/homework/config/WebSecurityConfig.java

```diff
@@ -47,6 +51,10 @@ public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
                                         .mvcMatchers(AUTH_WHITELIST)
                                         .permitAll()
                                         .mvcMatchers("/ads/**", "/users/**")
+                                        .hasAnyRole("USER", "ADMIN")
```

Comment:
```suggestion
                                        .mvcMatchers(AUTH_WHITELIST)
                                        .permitAll()
                                        .mvcMatchers(HttpMethod.GET,"/ads").permitAll()
                                        .mvcMatchers("/ads/**", "/users/**")
                                        .hasAnyRole("USER", "ADMIN")
```
Вот так можно дать доступ к списку объявлений